apps/roam/src/components/settings/QueryPagesPanel.tsx

```diff
@@ -46,6 +47,7 @@ const QueryPagesPanel = ({
             setTexts(newTexts);
             extensionAPI.settings.set("query-pages", newTexts);
             setValue("");
+            posthog.capture("Query builder Page: Created", { newType: value });
```

Comment:
This is recording a user adding another path to match which `[[query/pages]]` will render a query page.  We can remove this, I don't think we are very interested in it.

![image](https://github.com/user-attachments/assets/7b7b5d6d-3728-48c5-972d-8308b3717411)